### ISSUE TITLE
We need more people

### ISSUE THREAD

**Issue by pharpend** [learnmath/lysa#87]
We have maybe 5 contributors now. We need more people, especially authors & editors to review each other's work. Currently, it's left to @beingbrown , myself, and @jesyspa . It's taking about 24 hours for a patch to get reviewed and merged. That's way too long. We need that to be maybe 2 or 3 hours.

**Comment by DTSCode** [comment]
I would start contributing, but a) I don't have enough time between the site, nimbus, work, and family :\ and b) I have a very shallow knowledge of math. However, if you want to give me a list of sites, I can start errr advertising (for lack of a better term) on them using posts similar to what I did on reddit. Currently I have only posted that message on /r/learnprogramming and /r/learnmath

-----Original Message-----

**Comment by pharpend** [comment]
It might be a good idea to reach out to some of the writing-centric subreddits; some of the users might give us feedback on the writing style. I would also like to post on some of the math-centric subreddits.

I'm very loopy today, so let's revisit this issue tomorrow.

**Comment by DTSCode** [comment]
Fair enough. Until tomorrow then

-----Original Message-----

**Comment by ghost** [comment]
Because most of us have regular jobs, I think a 2-3 hour turnaround is unrealistic.

**Comment by wei2912** [comment]
We might want to advertise after we get a significant portion finished.

**Comment by wei2912** [comment]
This will be my favourite label.

**Comment by jesyspa** [comment]
Given that we are having trouble coordinating even the three of us, I doubt the problem is a lack of contributors.  Given that it would increase the number of people with feedback on each change, I expect more people to slow things down at this point, not speed them up.

**Comment by pharpend** [comment]
That's very true. @wei2912 I suggested a category of bugs called "pharpend is an asshole". This might be one of them.

@jesyspa you are right.

Nonetheless, we still need more people, so let's think about this.

**Comment by wei2912** [comment]
@pharpend Do you want that? I can create a label for `ministry of love`.

**Comment by wei2912** [comment]
Joking aside, I think that having 5 main contributors (and a few other contributors from the community) is a healthy number. For now, it should suffice. If we wish to get to 2-3 hours average review time, we'll need a very large and active community. It'll be hard to get more contributors for now, considering that it's not yet clear if this will continue for quite some time.

**Comment by pharpend** [comment]
That's very true. I'm still very loopy today, else I would probably post something to reddit.

**Comment by ghost** [comment]
I agree with @jesyspa and @wei2912

**Comment by jesyspa** [comment]
> @jesyspa you are right.
> Nonetheless, we still need more people, so let's think about this.

These two statements seem contradictory to me.  If I am right, why do we still need more people?

**Comment by pharpend** [comment]
@jesyspa right about not needing any more authors.

We should have more people in the community as a whole, if we are to gain any readership.

**Comment by DTSCode** [comment]
Basically, we need to a) grow bigger than #darkf, and b) get more feedback (I agree we don't nessecarily need more authors atm, but I don't see downsides to more editors). Essentially, we just need to advertise (particularly on reddit) more. I'll discuss this with @pharpend more when I get off of work 

-----Original Message-----

**Comment by ghost** [comment]
@DTSCode i'm less concerned about advertising now because we've got very little to show. I'd love it if we could inspire some more math folks around the channel, and if you'd like to pick up a second for the site, let me know. If you wouldn't, I'm closing this as resolved.

**Comment by wei2912** [comment]
I agree with @beingbrown. We've already posted a link on Reddit and managed to bring in a bit more traffic from other IRC channels. Beyond that, we'll need something to show to others. At least a significant portion of the book should be done.
